sdk/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Can you comment somewhere why is this needed? I am sure we will forget.

Reply:
I created a comment in `[Simple|Batch]SpanProcessorTest`

Reply:
@thisthat I'm guessing we can't create a comment right in this file?

Reply:
Unfortunately, according to the [documentation](https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods) it says "containing a single line" :(